packages/loot-core/src/shared/arithmetic.test.ts

```diff
@@ -41,4 +42,58 @@ describe('arithmetic', () => {
   test('respects current number format', () => {
     expect(evalArithmetic('1,222.45')).toEqual(1222.45);
   });
+
+  for (let format of ['dot-comma', 'comma-dot', 'space-comma', 'space-dot']) {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would you mind refactoring this to `test.each`? That's generally the way to use data-sets in unit tests.

Reply:
I pushed a refactor, please let me know if it's how you wanted it.

Reply:
Ok, this is slightly better, but we still have logic in unit tests (which is something we should ideally avoid). 

What I would recommend is extracting the permutation-building logic into a separate file (`mock-data.ts` / `fixtures.ts` or else) and work on removing the remaining `foreach` operation. The end goal would be - no `for` loops in test files (i.e. no logic, but _simple_ & _dumb_ test cases).

Reply:
Pushed.